src/main/java/de/hs_mannheim/informatik/ct/persistence/repositories/RoomRepository.java

```diff
@@ -45,4 +47,9 @@
     List<Room> findByBuildingName(String building);
 
 
+    @Query("SELECT SUM(room.maxCapacity) " +
```

Comment:
Hier bekomme ich einen Internal Server Error auf der Startseite, weil es bei mir keinen der Räume gibt. Gibt einen Error wenn man `SUM` auf ein leeres Ergebnis macht [(Link)](https://stackoverflow.com/a/10296943). Aber du solltest auch die Räume in der Application.properties so setzten das vllt der Test Raum ein Lernraum ist. Die defaults sind eigentlich immer so das es in der Dev Umgebung ohne Konfiguration läuft. Die tatsächlichen Lernräume kannst du dann in der `docker-compose.yml` überschreiben.